### PR TITLE
fix(tutorial): tiny improvements on weather warning application

### PR DESCRIPTION
**Pull request by hoelger**
## Description

Small fixes that arose when getting familiar with MOSAIC

## Issue(s) related to this PR

* Resolves internal issue 885
  
## Affected parts of the online documentation

Changes in the documentation required? No.

## Definition of Done

**Prerequisites**

- [x] You have read CONTRIBUTING.md carefully.
- [x] You have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [x] Your GitHub user id is linked with your Eclipse Account.

**Required**

- [x] The title of this merge request follows the scheme `type(scope): description`  (in the style of [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary))
- [ ] You have assigned a suitable label to this pull request (e.g., `enhancement`, or `bugfix`)
- [x] `origin/main` has been merged into your Fork.
- [x] Coding guidelines have been followed (see CONTRIBUTING.md).
- [x] All checks on GitHub pass.
- [x] All tests on [Jenkins](https://ci.eclipse.org/mosaic/job/mosaic/) pass.

**Requested** (can be enforced by maintainers)

- [ ] New functionality is covered by unit tests or integration tests. Code coverage must not decrease.
- [ ] If a bug has been fixed, a new unit test has been written (beforehand) to prove misbehavior
- [x] There are no new SpotBugs warnings.

### CODE REVIEW

**Comment by hoelger** [comment]
First finish the discussions in issue 885 and look if more code changes come up, then review and merge this PR

**Comment by kschrab** [comment]
@hoelger continous integration was not successful, check `BarnimReleaseIT`

File: app/tutorials/weather-warning/src/main/java/org/eclipse/mosaic/app/tutorial/WeatherWarningApp.java [hunk 1]
```diff
@@ -298,8 +297,8 @@ private void circumnavigateAffectedRoad(Denm denm, final String affectedRoadId)
          */
         CandidateRoute newRoute = response.getBestRoute();
         if (newRoute != null) {
-            getLog().infoSimTime(this, "Sending Change Route Command at position: {}", denm.getSenderPosition());
-            navigationModule.switchRoute(newRoute);
+            getLog().infoSimTime(this, "Sending Change Route Command at position: {}", getOs().getPosition());
+            routeChanged = navigationModule.switchRoute(newRoute);
```
[the remark this answers is not in the document]
Comment (hoelger) [reply app/tutorials/weather-warning/src/main/java/org/eclipse/mosaic/app/tutorial/WeatherWarningApp.java:301]: I see. I didn't think of it that way. For me then we could use the old structure but rename `routeChanged` to `triedToChangeRoute`. I just tried to correctly implement the variable as exact as possible to their semantic meaning.

[the remark this answers is not in the document]
Comment (kschrab) [reply app/tutorials/weather-warning/src/main/java/org/eclipse/mosaic/app/tutorial/WeatherWarningApp.java:301]: Understood. I actually like the idea of just renaming the field to `triedToChangeRoute`.

**Pull request merged**